src/components/FilterBadges/FilterBadge.tsx

```diff
@@ -1,32 +1,62 @@
 import styled from "@emotion/styled";
 import Badge from "@leafygreen-ui/badge";
 import { uiColors } from "@leafygreen-ui/palette";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { ConditionalWrapper } from "components/ConditionalWrapper";
 import Icon from "components/Icon";
+import { string } from "utils";
 
+const { trimStringFromMiddle } = string;
 const { gray } = uiColors;
 
+const tooltipInModalZIndex = 50; // necessary due to SeeMoreModal, which has zIndex 40
```

Comment:
yeah 😭 I was thinking about making a `constants/zindex.ts` file, but it'd be weird since all of the other `zIndex` are in a bunch of different files ....